Cargo.toml

```diff
@@ -112,6 +113,11 @@ name = "client"
 path = "examples/client.rs"
 required-features = ["full"]
 
+[[example]]
+name = "client_single_thread"
```

Comment:
Another option would be to make this part of the existing `single_threaded.rs` example, where it just both starts a server, and also makes use a of a client. That would mean less code that is copied and needs to be updated later on.

Reply:
Done. Do you have an idea why `io::stdout().write_all(&chunk).await?;` didn't work in the combined example? It just wrote nothing to console.

Reply:
fixed that in [improve example](https://github.com/hyperium/hyper/pull/3184/commits/3a78a57836417f19ca5b696ac7f3e42a2518a24d). forgot to mention it here.